index.html

```diff
@@ -212,7 +220,7 @@ <h2 class="about-title">About me</h2>
         </div>
 
         <div class="devider"></div>
-
+        
```

Comment:
- [ ]  About section is getting there, but we need to make some changes to match the Figma design:

![Captura de pantalla 2022-03-25 a la(s) 7 59 01 p  m](https://user-images.githubusercontent.com/83717285/160218800-03fe1586-654a-42ad-8734-245e70ffa92a.png)

- Please add some space on the left side of the About title and its paragraph.
- The about description should be split into several lines and narrower.
- Try centering the divider line and give it some space around.
- `Language, Framework and Skills` titles should be aligned horizontally.